server/api/rounds.py

```diff
@@ -120,6 +120,14 @@ def cumulative_contest_results(contest: Contest) -> Dict[str, int]:
     return results_by_choice
 
 
+# Get round-by-round audit results
+def contest_results_by_round(contest: Contest) -> Dict[str, Dict[str, int]]:
+    results_by_choice: Dict[str, Dict[str, int]] = defaultdict(lambda: defaultdict(int))
+    for result in contest.results:
+        results_by_choice[result.round_id][result.contest_choice_id] = result.result
+    return results_by_choice
```

Comment:
nit: i think `results_by_round` makes more sense than `results_by_choice` given the name of the function

Reply:
i think the change for this got left behind somewhere

Reply:
Fixed